apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -62,4 +66,41 @@ const getAuthenticatedUser = createAsyncThunk<
 	return await authApi.getAuthenticatedUser();
 });
 
-export { getAuthenticatedUser, logOut, signIn, signUp };
+const requestResetPassword = createAsyncThunk<
+	boolean,
+	EmailDto,
+	AsyncThunkConfig
+>(`${sliceName}/send-reset-password-link`, async (emailPayload, { extra }) => {
+	const { authApi, notification } = extra;
+
+	const isSuccesful = await authApi.requestResetPassword(emailPayload);
```

Comment:
```suggestion
	const isSuccessful = await authApi.requestResetPassword(emailPayload);
```
You can use spell checker extension to avoid such mistakes